PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,9 +26,11 @@ _ Add relevant screenshots_
 | --- | --- |
 | img | img |
 </details>
+
+
```

Comment:
```suggestion

```
Think it just needs one extra newline after the end tag of details

Reply:
Yeah, the suggestion was intended to remove one of the three newlines since only two are needed, but looks like in the commit that didn't happen right. Looks like you fixed it though : )